document/core/exec/instructions.rst

```diff
@@ -1885,6 +2000,103 @@ When the end of a block is reached without a jump or trap aborting it, then the
    Therefore, execution of a loop falls off the end, unless a backwards branch is performed explicitly.
 
 
+.. index:: exception handling, throw context
+   pair: handling; exception
+
+.. _exec-catch:
+
+Exception Handling
+~~~~~~~~~~~~~~~~~~
+
+The following auxiliary rules define the semantics of entering and exiting exception handlers through :ref:`try <syntax-try>` instructions and handling thrown exceptions.
+
+.. _exec-handler-enter:
+
+Entering an exception handler :math:`H`
+.......................................
+
+1. Push :math:`H` onto the stack.
+
+.. note::
+   No formal reduction rule is needed for installing an exception handler
+   because it is an :ref:`administrative instruction <syntax-instr-admin>`
+   that the :ref:`try <syntax-try>` instruction reduced to directly.
+
+.. _exec-handler-exit:
+
+Exiting an exception handler
+............................
+
+When the end of a :ref:`try <syntax-try>` instruction is reached without a jump, exception, or trap, then the following steps are performed.
+If an exception is thrown before the end of the |TRY| instruction, then we are in the case of :ref:`throwing <exec-throwaddr>` an exception.
```

Comment:
```suggestion
If an exception is thrown before the end of the |TRY| instruction, then the case of :ref:`throwing <exec-throwaddr>` an exception applies.
```
Or perhaps just remove the sentence -- it's kind of redundant, and we don't have such a note for trap either.

Reply:
sentence removed! :)